src/components/project-management-admin-panel.jsx

```diff
@@ -0,0 +1,270 @@
+  const [minimumVotes, setMinimumVotes] = useState(1);
+  const [positiveVotesPercentage, setPositiveVotesPercentage] = useState(50);
+  const [email, setEmail] = useState('');
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const roles = ["Project Manager", "Moderator", "Editor", "Voter", "Viewer"];
+
+  const handleRule1EndChange = (e) => {
+    setRule1End(e.target.value);
+  }
+
+  const handlerule1TimeframeChange = (e) => {
+    setRule1Timeframe(e.target.value);
+  }
+
+  const handleRule2StartChange = (e) => {
+    setRule2Start(e.target.value);
+  }
+
+  const handleRule2EndChange = (e) => {
+    setRule2End(e.target.value);
+  }
+
+  const handleRule2TimeframeChange = (e) => {
+    setRule2Timeframe(e.target.value);
+  }
+
+  const handleRule3StartChange = (e) => {
+    setRule3Start(e.target.value);
+  }
+
+  const handleRule3TimeframeChange = (e) => {
+    setRule3Timeframe(e.target.value);
+  }
+
+  const handleMinimumVotesChange = (e) => {
+    setMinimumVotes(e.target.value);
+  }
+
+  const handlePositiveVotesPercentageChange = (e) => {
+    setPositiveVotesPercentage(e.target.value);
+  }
+
+  const handleRoleChange = (email, newRole) => {
+    console.log(`Role for user ${email} changed to ${newRole}`);
+    updateUserRole(email, newRole);
+  };
+
+  function goToEditPage() {
+    console.log("Navigating to Edit Project page");
+    //TODO: Uncomment the following line after merge with edit page branch
+    //router.push("/collections/[name]/edit");
+  }
+
+  function saveChangeRuleChanges() {
+    console.log("Rule 1 start: ", rule1Start);
+    console.log("Rule 1 end: ", rule1End);
+    console.log("Rule 1 timeframe: ", rule1Timeframe);
+    console.log("Rule 2 start: ", rule2Start);
+    console.log("Rule 2 end: ", rule2End);
+    console.log("Rule 2 timeframe: ", rule2Timeframe);
+    console.log("Rule 3 start: ", rule3Start);
+    console.log("Rule 3 timeframe: ", rule3Timeframe);
+
+    const changeRulesData = {
+      rule1: {
+        rule1Start,
+        rule1End,
+        rule1Timeframe,
+      },
+      rule2: {
+        rule2Start,
+        rule2End,
+        rule2Timeframe
+      },
+      rule3: {
+        rule3Start,
+        rule3Timeframe
+      }
+    };
+
+    //TODO: Call API to save change rules
+  }
+
+  function saveVotingRuleChanges() {
+    console.log("Saving minimum votes: ", minimumVotes);
+    console.log("Saving positive votes percentage: ", positiveVotesPercentage);
+
+    const votingRulesData = {
+      minimumVotes,
+      positiveVotesPercentage
+    };
+
+    //TODO: Call API to save voting rules
+  }
+
+
+  function checkEmailAddress() {
+    console.log("Checking email address: ", email);
+    //TODO: Implement email address check
+    //TODO: If email is valid, set title and description with success message
+    //TODO: If email is invalid, set title and description with error message
+  }
```

Comment:
What's this do? If it adds a user to a project, it should be named accordingly like `sendInvite`, but this implies the user gets an email?